RELEASE.rst

```diff
@@ -0,0 +1,6 @@
+RELEASE_TYPE: patch
+
+This is a bugfix release for `~hypothesis.strategies.integers`.
+Given a ``min_value`` and ``max_value`` that have no integers between them,
+an internal assertion was triggered.  Non-integer bounds are still allowed,
+but InvalidArgument is now raised in this case.
```

Comment:
minor: the last sentence talks about “non-integer bounds”, but it’s not entirely obvious where that comes from – the second sentence doesn’t mention them. And I wasn’t worrying about them being taken away until you mentioned it!

Maybe this is clearer, what do you think?

> Previously the strategy would hit an internal assertion if passed non-integer bounds for ``min_value`` and ``max_value`` that had no integers between them. The strategy now raises ``InvalidArgument`` instead.